llm/app/api/visitorsbot_chat.py

```
@@ -0,0 +1,145 @@
+    try:
+        # 1. 이미 올바른 한글이면 그대로 반환
+        if all(ord(char) < 128 or '\uac00' <= char <= '\ud7af' for char in text if char.isalpha()):
+            return text
+
+        # 2. Latin-1로 잘못 인코딩된 UTF-8을 수정
+        try:
+            fixed = text.encode('latin-1').decode('utf-8')
+            return fixed
+        except (UnicodeDecodeError, UnicodeEncodeError):
+            pass
+        except (UnicodeDecodeError, ValueError):
```

Ruff (0.12.2)

try-except block with duplicate exception `UnicodeDecodeError`

(B025)